client/rpc-core/txpool/src/types/content.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-2020 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::GetT;
+use ethereum::{Transaction as EthereumTransaction, TransactionAction};
+use ethereum_types::{H160, H256, U256};
+use fc_rpc_core::types::Bytes;
+use serde::Serialize;
+
+#[derive(Debug, Default, Clone, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Transaction {
+	/// Hash
+	pub hash: H256,
+	/// Nonce
+	pub nonce: U256,
+	/// Block hash
+	pub block_hash: Option<H256>,
+	/// Block number
+	pub block_number: Option<U256>,
+	/// Sender
+	pub from: H160,
+	/// Recipient
+	pub to: Option<H160>,
+	/// Transfered value
+	pub value: U256,
+	/// Gas Price
+	pub gas_price: U256,
+	/// Gas
+	pub gas: U256,
+	/// Data
+	pub input: Bytes,
+}
+
+impl GetT for Transaction {
+	fn get(hash: H256, from_address: H160, txn: &EthereumTransaction) -> Self {
+		Self {
+			hash,
+			nonce: txn.nonce,
+			block_hash: Some(H256::default()), // or None?
```

Comment:
`None` is a reasonable default if this field has not yet been set. 

If the block hash was ever needed and this field was `None`, then you could `unwrap_or(|| H256::default())` like you did for the `to` field in the `Serialize` implementation of `Summary`

Reply:
Thank you! Just added it as well as a serializer for the Transaction's `to` field.